daemon/execdriver/driver.go

```diff
@@ -24,10 +24,21 @@ var (
 	ErrDriverNotFound          = errors.New("The requested docker init has not been found")
 )
 
-// StartCallback defines a callback function.
-// It's used by 'Run' and 'Exec', does some work in parent process
-// after child process is started.
-type StartCallback func(*ProcessConfig, int)
+// DriverCallback defines a callback function which is used in "Run" and "Exec".
+// This allows work to be done in the parent process when the child is passing
+// through PreStart, Start and PostStop events
+type DriverCallback func(*ProcessConfig, int) error
```

Comment:
Can you assign a var name to `int` or document what `int` is expected to be here?